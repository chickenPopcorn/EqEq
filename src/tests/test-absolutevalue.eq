SomeCtx = {
  a = { -52; }
}
SomeCtx:find a {
  print("the value of a is %.0f\n", a );
  print("the value of a %% 3 is %.0f\n", a % 3 );
  print("the value of | a %% 3 | is %.0f\n", | a % 3 | );
  print("the value of | a %% 3 | + 2 is %.0f\n", | a % 3 | + 2 );
  print("the value of - ( | a %% 3 | + 2 ) is %.0f\n", - ( | a % 3 | + 2 ) );
}