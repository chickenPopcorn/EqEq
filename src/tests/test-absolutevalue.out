the value of a is -52
the  value of a % 3 is -1
the  value of | a % 3 | is 1
the value of | a % 3 | + 2 is 3
the value of - ( | a % 3 | + 2 ) is -3
